apps/backend/cypress/e2e/project.cy.ts

```diff
@@ -1,77 +1,96 @@
+import { ProjectDto } from '@datatlas/shared/models';
+
 describe('PROJECT ACTIONS', () => {
-  const test_project = {
-    title: 'project_test',
-    description: 'project_test_description',
+  const test_project = new ProjectDto({
+    title: 'titre projet test 2',
     draft: true,
-    datasets: 'tutu',
-    owner: 'possesseur',
-    contributors: 'contributeurs',
-    config: 'config',
-    version: 'versionNumber',
-    createdAt: 'dateCreation',
-  };
-  const modified_test_project = {
-    title: 'modified_project_test',
-    description: 'modified_project_test_description',
-    draft: true,
-    datasets: 'toto',
-    owner: 'possesseur',
-    contributors: 'contributeurs',
-    config: 'config',
-    version: 'versionNumber',
-    createdAt: 'dateCreation',
-  };
-  let id_test_project = null;
-  it('Project -> creation of new project (should return new project id)', () => {
+    datasets: { toto: 'test 2' },
+    description: 'description du projet 2',
+    config: { toto: 'test 2' },
+    version: 'version test',
+    contributors: [63, 64], // Shall we really send IDs ? Maybe usernames instead ?
+  });
+  const test_project_modified = new ProjectDto({
+    title: 'titre projet test 2_',
+    draft: false,
+    datasets: { toto: 'test 2_' },
+    description: 'description du projet 2_',
+    config: { toto: 'test 2_' },
+    version: 'version test_',
+    contributors: [63], // Shall we really send IDs ? Maybe usernames instead ?
```

Comment:
Why `createdAt` or `modifiedAt` are not kept ?

Reply:
They are kept. It's just the backend that determines them automatically (no use for the front to send them).

Reply:
We clearly need to discuss about these topics.

Reply:
#128